doc/design/refactorization.md

```diff
@@ -136,15 +136,15 @@ Compile Time -> IR -> Runtime
 * `Eigen::Tensor` contains basic math and element-wise functions.
     * Note that `Eigen::Tensor` has broadcast implementation.
     * Limit number of `tensor.device(dev) = ` in your code.
-* `thrust::tranform` and `std::transform`.
-    * `thrust` has the same API as C++ standard library. Using `transform` can quickly implement a customized elementwise kernel.
-    * `thrust` has more complex API, like `scan`, `reduce`, `reduce_by_key`.
+* `thrust::transform` and `std::transform`.
+    * `thrust` has the same API as C++ standard library. Using `transform` can quickly implement a customized element wise kernel.
```

Comment:
Maybe element-wise is correct? anyway, it doesn't affect the meaning. They use the same word in these famous opensource projects. :)
[pandas](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.map.html)
[scipy](http://www.scipy-lectures.org/intro/numpy/operations.html)